system/config/templates/features/enroll_app/enroll_app.yml

```diff
@@ -308,6 +308,8 @@ registry:
             item: 85
           - key: :child_age_limit
             item: 20
+          - key: :default_password_length_range
+            item: 8..20
```

Comment:
why we dont have this from enviroment variable, that way its visible; somehting like this;

item: <%= ENV['PASSWORD_LENGHT_RANGE'] || "8..20" %>